### PR TITLE
New package: jgmenu

### PR DESCRIPTION
**Pull request by cr6git**
Closes [#7004](https://github.com/voidlinux/void-packages/issues/7004).

### CODE REVIEW

File: srcpkgs/jgmenu/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+pkgname=jgmenu
+version=0.6.1
+revision=1
+build_style=gnu-makefile
+make_use_env=yes
+make_install_args="prefix=/usr"
+hostmakedepends="pkg-config python3"
+makedepends="cairo-devel libglib-devel librsvg-devel libxml2-devel
+ libX11-devel libXinerama-devel menu-cache-devel pango-devel"
+depends="python3"
```
Comment (ebfe) [inline srcpkgs/jgmenu/template]: Where does the python3 dependency come from?

Comment (cr6git) [inline srcpkgs/jgmenu/template]: You are right! python3 is not needed, I don't remember why I added it.

**Comment by cr6git** [comment]
@ebfe It's ok for you ?

File: srcpkgs/jgmenu/template (hunk 1 above)
Comment (ebfe) [inline srcpkgs/jgmenu/template]: There are python 2 scripts in usr/lib/jgmenu/ and a python script that's run during build in scripts/. Sorry, inital version was right.

Comment (cr6git) [inline srcpkgs/jgmenu/template]: I trust you. I'm glad to see this package accepted.

**Pull request closed without merging**